apps/src/dance/Dance.js

```diff
@@ -93,6 +93,7 @@ Dance.prototype.init = function (config) {
   }
 
   this.level = config.level;
+  this.usesPreview = !!config.level.usesPreview;
```

Comment:
Sorry I didn't notice earlier, but it seems that level properties generally use `snake_case` rather than `camelCase`?   If so, we should probably be consistent.

Reply:
We also snapshot `config.level` into `this.level`, so we could use `this.level.uses_preview` in the rest of this file rather than snapshotting this value separately.